### PR TITLE
fix: In the trace panel, select service from the drop-down box, and click invalid when search

### PR DESCRIPTION
**Pull request by jianglin1008**
#521 
image

### CODE REVIEW

**Comment by wu-sheng** [comment]
@jianglin1008 Is this also caused by duplicate keys?

File: src/views/components/common/common-selector.vue [hunk 1]
```diff
@@ -60,8 +60,8 @@ limitations under the License. -->
           class="rk-common-opt ell"
           @click="handleSelect(i)"
           :class="{ active: i.key === value.key }"
-          v-for="i in filterData"
-          :key="i.key"
+          v-for="(i, idx) in filterData"
+          :key="idx"
```
Comment (wu-sheng) [inline src/views/components/common/common-selector.vue:64]: Does this effect the query condition?

**Comment by jianglin1008** [comment]
> @jianglin1008 Is this also caused by duplicate keys?

yes, it is

File: src/views/components/common/common-selector.vue (hunk 1 above)
Comment (jianglin1008) [reply src/views/components/common/common-selector.vue:64]: No. The query uses the var i.

**Review by Fine0830 — APPROVED** [review]

**Comment by wu-sheng** [comment]
@jianglin1008 Could you update UI sub module of main repo?

**Comment by jianglin1008** [comment]
@wu-sheng PR has been created.

**Pull request merged**